File: sdkwork-ai-sdk-csharp/Models/JsonSchema.cs
using System;
using System.Collections.Generic;
using System.Text.Json.Serialization;

namespace Ai.Models
{
    public class JsonSchema
    {
        public string? Name { get; set; }
        public string? Description { get; set; }
        public object? Schema { get; set; }
        public bool? Strict { get; set; }
    }
}

File: sdkwork-ai-sdk-typescript/src/types/json-schema.ts
/** JSON Schema配置 */
export interface JsonSchema {
  /** JSON结构名称 */
  name?: string;
  /** 描述 */
  description?: string;
  /** JSON Schema对象 */
  schema?: unknown;
  /** 是否严格遵循 */
  strict?: boolean;
}
